src/GuestBook/RegularGuest.java

```diff
@@ -0,0 +1,19 @@
+package GuestBook;
+
+public class RegularGuest implements Guest {
+	
+	private String name;
+
+	public RegularGuest(String name) {
+		this.name = name;
+		
+	}
+	
+	
+	public String getListing() {
+		
+		return name;
+	}
+
```

Comment:
maybe consider adding a toString? There's one on VIPGuest but not on RegularGuest